providers/multipart/src/main/java/org/jboss/resteasy/plugins/providers/multipart/MultipartInputImpl.java

```diff
@@ -507,131 +383,4 @@ public void setProviders(Providers providers)
    {
       savedProviders = providers;
    }
-
-   /**
-    * A custom TempFileStorageProvider that do no set deleteOnExit on temp files,
-    * to avoid memory leaks (see https://issues.apache.org/jira/browse/MIME4J-251)
```

Comment:
Note this jira is still open, so we probably still need to be careful and provide a workaround for it